src/pages/components/menu-buttons/accessibility.mdx

```diff
@@ -208,4 +208,7 @@ custom component.
 - When the combo button menu is open, the element (`div`) wrapping the entire
   combo button is given `aria-owns` with a value of the menu `id`.
 
-<A11yStatus layout="table" components="Menu buttons" />
+<A11yStatus
+  layout="table"
+  components={['Menu buttons', 'Combo button', 'Overflow menu']}
```

Comment:
```suggestion
  components={['Menu button', 'Combo button', 'Overflow menu']}
```

Reply:
I already did that but there is no component called menu button so its not even getting rendered in the table thats what I was not sure about either I should rename to component in carbon website or I should do that CUSTOM LOGIC , as I think renaming the component might break many things in website.

Reply:
Ah I think I know what's missing. This needs to get added to the data/components.json for Menu Button to show up. 

```
{
  "component": "Menu button",
  "parentComponent": "Menu buttons",
  "overview": false,
  "testing": {
    "screenreader": "manual"
  }
},
```